test/libsolidity/semanticTests/externalContracts/prbmath_unsigned.sol

```diff
@@ -0,0 +1,95 @@
+==== ExternalSource: _prbmath/PRBMathCommon.sol ====
+==== ExternalSource: _prbmath/PRBMathUD60x18.sol ====
+==== Source: prbmath.sol ====
+import "_prbmath/PRBMathUD60x18.sol";
+
+contract test {
+    using PRBMathUD60x18 for uint256;
+
+    function div(uint256 x, uint256 y) external pure returns (uint256 ret) {
+        ret = x.div(y);
+    }
+    function exp(uint256 x) external pure returns (uint256 ret) {
+        ret = x.log10();
+    }
+    function exp2(uint256 x) external pure returns (uint256 ret) {
+        ret = x.exp2();
+    }
+    function gm(uint256 x, uint256 y) external pure returns (uint256 ret) {
+        ret = x.gm(y);
+    }
+    function log10(uint256 x) external pure returns (uint256 ret) {
+        ret = x.log10();
+    }
+    function log2(uint256 x) external pure returns (uint256 ret) {
+        ret = x.log2();
+    }
+    function mul(uint256 x, uint256 y) external pure returns (uint256 ret) {
+        ret = x.mul(y);
+    }
+    function pow(uint256 x, uint256 y) external pure returns (uint256 ret) {
+        ret = x.pow(y);
+    }
+    function sqrt(uint256 x) external pure returns (uint256 ret) {
+        ret = x.sqrt();
+    }
+    function benchmark(uint256 x) external pure returns (uint256 ret, uint256 z1, uint256 z2) {
+        uint256 y = x.div(3).ceil();
+        uint256 z = x.mul(y);
+        for (uint i = 0; i < 10; i++)
+            z = z.sqrt();
+        ret = z;
+
+        // Check precision
+        z1 = z.ceil();
+        z2 = z.sqrt().pow(2).ceil();
+        assert(z1 == z2);
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// constructor()
+// gas irOptimized: 1937802
+// gas legacy: 2326345
+// gas legacyOptimized: 1750080
+// div(uint256,uint256): 112234234324234, 88714123 -> 1265122514080807629693865
+// gas irOptimized: 22079
+// gas legacy: 22473
+// gas legacyOptimized: 21986
+// exp(uint256): 112234234324234 -> FAILURE
+// gas irOptimized: 21683
+// gas legacy: 21758
+// gas legacyOptimized: 21575
+// exp2(uint256): 112234234324234 -> 1000077797869181423
+// gas irOptimized: 24788
+// gas legacy: 24865
+// gas legacyOptimized: 24080
+// gm(uint256,uint256): 112234234324234, 88714123 -> 99783574142
+// gas irOptimized: 23519
+// gas legacy: 23184
+// gas legacyOptimized: 22643
+// log10(uint256): 112234234324234 -> FAILURE
+// gas irOptimized: 21727
+// gas legacy: 21802
+// gas legacyOptimized: 21619
+// log2(uint256): 112234234324234 -> FAILURE
```

Comment:
Haven't investigated why these fail. Should probably select a better input number for these tests. Will go with what `pi()` returns.